style.css

```diff
@@ -0,0 +1,448 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+a {
+  font-weight: 100;
+  letter-spacing: 2px;
+  color: wheat;
+  text-decoration: none;
+  padding: 20px 5px;
+  width: 100%;
+  display: inline-block;
+  text-align: center;
+  transition: all 0.5s;
+}
+
+li {
+  list-style: none;
+}
+
+.wrapper {
+  width: 375px;
+  margin: 0 auto;
+  height: fit-content;
+  display: flex;
+  background-color: #e5e5e5;
+  flex-direction: column;
+}
```

Comment:
Kindly remove the grey background color in this header in other to follow the Figma design 😊 ☺️ 🟢 . 

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="373" alt="image" src="https://user-images.githubusercontent.com/73854999/219463214-709e1164-0447-418b-a8c9-cae55e8314ca.png"> | <img width="373" alt="image" src="https://user-images.githubusercontent.com/73854999/219462752-2c6941cb-a7d5-4446-9125-93657310c022.png">|